### PR TITLE
Packaging standards

### PR DESCRIPTION
**Pull request by dvzrv**
content/pages/report.md: Adding formatted overview of what was discussed during arch conf on the topic of package standards/ packaging guidelines/ best practices/ problems.

### CODE REVIEW

File: content/pages/report.md [hunk 1]
```diff
@@ -127,7 +127,64 @@ terraform now it would be logical manage our DNS also with terraform.
...
+
+Additionally, issues that come up during TU application reviews are a good
+example of inconsistencies, that are worth documenting right away.
+
+The following list is a (non-exhaustive) collection of examples, that still
+require addition in the wiki.
+
+- use HTTPS for sources whereever possible, "git+https://" for git
+- don't use specific sourceforge mirror to download:
+  https://bbs.archlinux.org/viewtopic.php?id=22200
```
Comment (jelly) [inline content/pages/report.md]: nitpick, I believe we have a way to make links ;-) []() ??

File: content/pages/report.md [hunk 2]
```diff
@@ -127,7 +127,64 @@ terraform now it would be logical manage our DNS also with terraform.
...
+  source tarball)
+- rename sources to something unique for shared `$srcdir`
+- be consistent with the use of cd "$srcdir/..." (which is not needed anyway)
+  or use subshells `( )`
+- don't use internal functions like `msg2` as they are subject to change and
+  *will break*
+- `autoreconf`, `autogen.sh`, `bootstrap`, etc. needs to be called in
+  `prepare()`, not in  `build()`
+- always do `autoreconf -vfi` (instead of e.g. `autogen.sh` or directly using a
+  configure file), so that system flags are honored
```
Comment (jelly) [inline content/pages/report.md]: Does this fix for example getting FULL RELRO? @anthraxx

**Comment by jelly** [comment]
Looks good to me! Minor nitpicks only really :)

File: content/pages/report.md (hunk 2 above)
Comment (dvzrv) [reply content/pages/report.md]: IIRC this is important so certain vendored libtool stuff doesn't ignore us setting e.g. `--as-needed` in LDFLAGS.

**Pull request merged**